src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -303,8 +352,8 @@ def main():
     parser_run.add_argument('--config', default='adam_preprocessing.config', type=str,
                             help='Path to the (filled in) config file, generated with "generate-config". '
                                  '\nDefault value: "%(default)s"')
-    parser_run.add_argument('--sample', help='The full s3 url of the input SAM or BAM file')
-    parser_run.add_argument('--output-dir', default=None,
+    parser_run.add_argument('--sample', help='The full s3 url/path to the input SAM or BAM file')
```

Comment:
This is a bit different from the other paths, since we do the download using conductor.